scripts/buildProtocol.ts

```diff
@@ -48,7 +48,8 @@ class DeclarationsWalker {
         }
         if (s.name === "Array" || s.name === "ReadOnlyArray") {
             // we should process type argument instead
-            return this.processType((<any>type).typeArguments[0]);
+            const typeArgs = this.typeChecker.getTypeArguments(type as ts.TypeReference);

```

Comment:
Ideally we'd probably also check if `type.flags & ts.TypeFlags.TypeReference` just to be sure, but I don't think we'll do something stupid like `enum Array {}` anywhere, so it's likely safe to assume it's a `TypeReference`.